temporalio/bridge/src/runtime.rs

```diff
@@ -118,12 +118,31 @@ pub fn init_runtime(telemetry_config: TelemetryConfig) -> PyResult<RuntimeRef> {
         }
     }
 
+    let task_locals = Python::with_gil(|py| {
+        // Event loop is assumed to be running at this point
```

Comment:
Just realized, I don't think we can assume this. People set global Runtime maybe before they've even done `asyncio.run`

Reply:
Yeah, good point, I'll try to move this to worker validate or run